includes/admin/class-wc-rest-payments-terminal-locations-controller.php

```diff
@@ -162,7 +162,22 @@ public function get_store_location( $request ) {
 
 			return rest_ensure_response( $this->extract_location_fields( $location ) );
 		} catch ( API_Exception $e ) {
-			return rest_ensure_response( new WP_Error( $e->get_error_code(), $e->getMessage() ) );
+			$error = new WP_Error( $e->get_error_code(), $e->getMessage() );
+			if ( 'city_invalid' === $e->get_error_code() || 'state_invalid' === $e->get_error_code() ) {
```

Comment:
The approach looks good to me! It looks like the thought that we have to match on `"param": "address[state/city/...]"`?